### DISCUSSION TITLE
[Feature] Add Text Notes to Photo Groups

### DISCUSSION THREAD

**Discussion by zhangboyi** [immich-app/immich#13634] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Introduce a feature that allows users to add text notes to a group of photos. These notes can be memories, emotions, or brief descriptions, helping users not only recall the visual aspects of the moment but also deepen their connection to the memory with written context.

Feature Benefits:
1. Enhanced Memory: By combining text notes with photos, users can more fully recall the circumstances and emotions tied to those moments.
2. Personalized Experience: The text adds emotional depth and background to the photos, creating a more personalized and meaningful memory.
3. Sharing and Interaction: Users can share photo groups with notes, enabling friends and family to relive memories together, fostering more interaction and connection.

功能请求：为照片组添加文字备注功能

在照片组中增加一个功能，允许用户为一组照片添加文字备注。这些文字可以是回忆、感受或简短的描述，帮助用户在未来查看这些照片时，不仅能通过视觉回忆当时的场景，还能通过文字的帮助加深记忆与情感连接。

功能意义：
1. 增强记忆：通过文字备注与照片的结合，用户能更全面地回忆当时的情景与感受。
2. 个性化体验：文字备注为照片赋予更多情感和背景信息，创造更深刻的个性化回忆。
3. 共享与互动：用户可以通过分享带备注的照片组，与家人朋友一起重温回忆，提供更多互动性。

### Platform

- [x] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
At the top of an album you can add a description

![image](https://github.com/user-attachments/assets/1c5760a2-e6e7-451c-9649-7020b184f24c)
